services/ui/Dockerfile

```
@@ -6,11 +6,21 @@ COPY package*.json ./
 RUN npm install
 
 COPY . .
-RUN npm run build \
-    && cp -rv node_modules/@uswds/uswds/dist/fonts build/_app/assets/
+RUN PORT=80 npm run build \
+    && cp -rv node_modules/@uswds/uswds/dist/fonts build/client/_app/assets/
 
-FROM nginx:1.21
-COPY --from=build /home/node/app/build /usr/share/nginx/html
+FROM node:16.15.0
```

Looks like the `node:16.15.0` image packages an older version of `npm` with it. It's fixed in `node:16.15.1`. It might make sense to rely on the `node:16.15` tag so we get point releases.

```suggestion
FROM node:16.15
```